File: CodeKata/Looping - 14/Python/943. Even and Odd Digits.py
v = input().strip()
even_digits = []
odd_digits = []

for i in v:
    if int(i) % 2 == 0:
        even_digits.append(i)
    else:
        odd_digits.append(i)

even_digits.sort()
odd_digits.sort()
print(" ".join(even_digits))
print(" ".join(odd_digits))

File: CodeKata/Looping - 14/JavaScript/943. Even and Odd Digits.js
const readline = require("readline");

const inp = readline.createInterface({
    input: process.stdin
});

const userInput = [];

inp.on("line", (data) => {
    userInput.push(data);
});

inp.on("close", () => {

    let even = []
    let odd = []
    let vp = userInput[0].split("")

    for (let i of vp) {
        if (+i % 2 == 0) {
            even.push(i)
        }
        else {
            odd.push(i)
        }
    }
    even.sort()
    odd.sort()
    console.log(even.join(" "))
    console.log(odd.join(" "))

});